charts/kcp-operator/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for kcp-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+crds:
+  enabled: true
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
```

Comment:
This is missing some punctuation not a sentence.

Reply:
Fixed.